From 62ef2029a0bc7a6938a064e90b2b07eec9d7b314 Mon Sep 17 00:00:00 2001
From: Daniel Ammann <daniel.ammann@bytesatwork.ch>
Date: Fri, 27 Aug 2021 17:03:56 +0200
Subject: [PATCH] ARM: dts: stm32: Fix issue with usart1 when using no-scmi

Add a reset to usart1 for consistency. Without it, it won't compile.

Signed-off-by: Daniel Ammann <daniel.ammann@bytesatwork.ch>
---
 arch/arm/boot/dts/stm32mp15-no-scmi.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/stm32mp15-no-scmi.dtsi b/arch/arm/boot/dts/stm32mp15-no-scmi.dtsi
index 94a10b86a179..4939f96da739 100644
--- a/arch/arm/boot/dts/stm32mp15-no-scmi.dtsi
+++ b/arch/arm/boot/dts/stm32mp15-no-scmi.dtsi
@@ -154,4 +154,5 @@
 
 &usart1 {
 	clocks = <&rcc USART1_K>;
+	resets = <&rcc USART1_R>;
 };
-- 
2.20.1

